core/serialisation/data_serialiser.py

```diff
@@ -44,6 +44,24 @@
 )
 
 
+class PostcodeList(fields.Field):
+    """Used to convert between a postcode array (as stored in the DB) and a CSV string (as appears in json/excel)"""
+
+    def _serialize(self, value, attr, obj, **kwargs):
```

Comment:
I think I mentioned this before, but it may have got lost in the comments or something:

Please could you add a test to test_serialisation, that specifically looks for a multi-postcode row in ProjectDetails, and asserts it's format. 
running debugger on `tests.serialisation_tests.test_serialisation.test_serialise_download_data_no_filters` should give you a clue how to implement (row 05 of that dict has 2 postcodes)